### PR TITLE
Change API acronym to uppercase

### PR DESCRIPTION
**Pull request by leemaguire**
Remove any lowercase acronym of API

### CODE REVIEW

File: Realm/RLMAPIKeyAuth.h [hunk 1]
```diff
@@ -40,7 +40,7 @@ typedef void(^RLMUserAPIKeysBlock)(NSArray<RLMUserAPIKey *> *  _Nullable, NSErro
   @param name The name of the API key to be created.
   @param completion A callback to be invoked once the call is complete.
 */
-- (void)createApiKeyWithName:(NSString *)name
+- (void)createAPIKeyWithName:(NSString *)name
                   completion:(RLMOptionalUserAPIKeyBlock)completion NS_SWIFT_NAME(createApiKey(named:completion:));
```
Comment (pavel-ship-it) [inline Realm/RLMAPIKeyAuth.h:44]: Should we change Swift name too?

Comment (leemaguire) [reply Realm/RLMAPIKeyAuth.h:44]: true

Comment (jsflax) [reply Realm/RLMAPIKeyAuth.h:44]: Im not sure these are extended in Swift until Pavel's Combine PR.

Comment (pavel-ship-it) [reply Realm/RLMAPIKeyAuth.h]: Yes, this is available in both callback and future forms

**Review by jsflax — APPROVED** [review]
Make sure tests are compiling before merging 😅 . As far as passing, I have a number of fixes in my branch that will cover us.

**Pull request merged**